addons/udes_stock/models/stock_move_line.py

```diff
@@ -173,9 +173,10 @@ def get_quants(self):
 
         return quants
 
-    def reset_move_line_data(self, default_location_dest):
+    def reset_move_line_data(self):
         """Resets the move_line data for location destination, quantity done, done datetime and
          done status to default values."""
+        default_location_dest = self.u_picking_type_id.default_location_dest_id
```

Comment:
This will raise error in case that move lines will be off different picking type. We might reuse this method in future and would be good to add a ensure_one for the picking type so we know that we need to use this method when resetting move lines of same picking type.

self.u_picking_type_id.ensure_one()